packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -655,16 +655,39 @@
   }
 }
 
-.mdc-text-field--box:not(.mdc-text-field--upgraded) {
-  height: 56px;
+.mdc-text-field:not(.mdc-text-field--upgraded) {
+  &.mdc-text-field--outlined {
```

Comment:
Is there a reason you nested these selectors and increased their specificity?

Would this work just as well as `.mdc-text-field--box:not(...)` and `.mdc-text-field--outlined:not(...)`?

Reply:
`.mdc-text-field--outlined:not(...)` wasn't strong enough to override the `.mdc-text-field:not(.mdc-text-field--upgraded):not(.mdc-text-field--textarea) .mdc-text-field__input` selector from above, so I removed `:not(.mdc-text-field--textarea)` from that and put in its own rule.

So now we have 
- `.mdc-text-field--outlined:not(...)`
- `.mdc-text-field--box:not(...)`
- `.mdc-text-field--textarea:not(...)`